service-tests/github.js

```diff
@@ -447,3 +447,31 @@ t.create('github pull request check state')
 t.create('github pull request check contexts')
   .get('/status/contexts/pulls/badges/shields/1110.json')
   .expectJSONTypes(Joi.object().keys({ name: 'checks', value: '1 failure' }));
+
+t.create('top language')
+.get('/languages/top/badges/shields.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('JavaScript'),
+  value: Joi.string().regex(/^([1-9]?[0-9]\.[0-9]|100\.0)%$/),
+}));
+
+t.create('top language with empty repository')
+.get('/languages/top/pyvesb/emptyrepo.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('language'),
+  value: Joi.equal('none'),
+}));
```

Comment:
You can tighten this up a bit since these are literals:
```js
.expectJSON({ name: 'language', value: 'none' })
```

Reply:
Fixed.